### PR TITLE
FEATURE: [datasource] add wise rate API

### PR DESCRIPTION
**Pull request by narumiruna**
https://docs.wise.com/api-docs/api-reference/rate

### CODE REVIEW

File: pkg/datasource/wise/rates_request.go [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+// https://docs.wise.com/api-docs/api-reference/rate
+
+//go:generate requestgen -method GET -url "/v1/rates" -type RatesRequest -responseType []Rate
+type RatesRequest struct {
+	client requestgen.AuthenticatedAPIClient
+
+	source string  `param:"source"`
+	target string  `param:"target"`
+	time   *string `param:"time"`
```
Comment (c9s) [inline pkg/datasource/wise/rates_request.go:15]: timeFormat attribute is supported, you can use that

Comment (c9s) [reply pkg/datasource/wise/rates_request.go:15]: image

Comment (narumiruna) [reply pkg/datasource/wise/rates_request.go:15]: Wise use different time format `2006-01-02T15:04:05-0700` XD

Comment (c9s) [reply pkg/datasource/wise/rates_request.go:15]: You can customize the format~

Comment (narumiruna) [reply pkg/datasource/wise/rates_request.go]: ok

Comment (c9s) [reply pkg/datasource/wise/rates_request.go]: You can set something like

timeFormat:"2006-01-02T15:04:05-0700"

Comment (narumiruna) [reply pkg/datasource/wise/rates_request.go]: done

**Review by c9s — APPROVED** [review]

**Pull request merged**